internal/agent/device/bootstrap.go

```diff
@@ -201,9 +206,15 @@ func (b *Bootstrap) ensureBootedOS(ctx context.Context, desired *v1alpha1.Render
 		return fmt.Errorf("writing current rendered spec: %w", err)
 	}
 
+	bootcStatus, err := b.bootcClient.Status(ctx)
```

Comment:
as a followup in agent.go when we create the bootc client lets fail fast if bootc does not exist.